docker/templates/macros.m4

```diff
@@ -23,9 +23,9 @@ copy_mb(``package.json yarn.lock ./'')
 RUN apt-key add /tmp/yarn_pubkey.txt && \
     rm /tmp/yarn_pubkey.txt && \
     echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
-    apt_install(``git python-minimal yarn'') && \
+    apt_install(``git python3-minimal yarn'') && \
```

Comment:
That has been added in commit b68b7b853e8abd82f41a1c0eabc09a3ef48afb32. Is it still required nowadays?

Reply:
Yes, nodejs still depends on it: https://deb.nodesource.com/node_16.x/dists/focal/main/binary-amd64/Packages